README.markdown

```diff
@@ -55,17 +52,37 @@ Rack::Timeout.timeout = 5
 ```ruby
 # config.ru
 require "rack-timeout"
-use Rack::Timeout          # Call as early as possible so rack-timeout runs before all other middleware.
-Rack::Timeout.timeout = 5  # Recommended. If omitted, defaults to 15 seconds.
+
+# Call as early as possible so rack-timeout runs before all other middleware.
+# Setting service_timeout is recommended. If omitted, defaults to 15 seconds.
+use Rack::Timeout, service_timeout:5
 ```
 
 
 The Rabbit Hole
 ---------------
 
+Rack::Timeout has four settings, with the following defaults:
```

Comment:
Another nit, but including changeable values like "four" in the preamble text doesn't add much, but tends to make it more prone to problematic drifts. i.e. A new setting is added, but someone forgets to bump the "four" to a "five". I'd probably go with something more like "Rack::Timeout settings have the following defaults:"